terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/login/LoginController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2013-2016 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.tourreservation.app.login;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/login")
+public class LoginController {
+
+    /**
+     * forward login view.
+     * 
+     * @return login view
+     */
+    @RequestMapping
+    public String login() {

```

Comment:
@kazuki43zoo 
Right, the name of handler that show input form is `xxxForm`.
`loginForm` is better than `login`, Thanks.